include/crow/app.h

```diff
@@ -360,7 +360,7 @@ namespace crow
     private:
         uint16_t port_ = 80;
         uint16_t concurrency_ = 1;
-        std::string server_name_ = "Crow/0.3";
+        std::string server_name_ = "Crow/master";
```

Comment:
Maybe create new header with some constexpr `VERSION_STRING` and use it here and in `http_server.h`. And then change this version from `master` to appropriate in cmake script on `deploy`/`package` target? I could prepare something like this maybe tomorrow if you want :)

Reply:
That sounds great